.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,93 @@
+name: Integration Tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  es-integration-tests:
```

Comment:
They all look quite similar. If they are indeed the same, just changing a parameter or two, then it might be better to use matrix jobs: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix